### PR TITLE
Collapse Release 1.1.0 into master

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
All Minitests pass on n30k, n31k, n9k; with exceptions due to the known issue with the sample rpm.

Maintainers/Contributing guides followed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,28 +4,53 @@ Cisco Network Elements support a rich set of features to make networks robust, e
 ## Getting Started
 
 * Create a [GitHub account](https://github.com/signup/free)
-* Create a [cisco.com](http://cisco.com) account if you need access to a Network Simulator to test your code.
+* N9kv. A virtual Nexus N9000/N3000 may be helpful for development and testing. Users with a valid [cisco.com](http://cisco.com) user ID can obtain a copy of a virtual Nexus N9000/N3000 by sending their [cisco.com](http://cisco.com) user ID in an email to <get-n9kv@cisco.com>. If you do not have a [cisco.com](http://cisco.com) user ID please register for one at [https://tools.cisco.com/IDREG/guestRegistration](https://tools.cisco.com/IDREG/guestRegistration)
```
Comment (glennmatthews) [inline CONTRIBUTING.md]: Looks a bit odd:
- N9kv. A virtual Nexus N9000/N3000 may...

Should the 'N9kv. ' be removed?

**Comment by glennmatthews** [comment]
Possibly useful: https://github.com/cisco/cisco-network-node-utils/compare/develop...release_1.1.0

**Comment by glennmatthews** [comment]
:+1:

File: CONTRIBUTING.md (hunk 1 above)
Comment (chrisvanheuveln) [inline CONTRIBUTING.md]: It's a bullet title. But I'll remove it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,8 +84,15 @@ Changelog
...
------
+## 0.9.0
 
 * First public release, corresponding to Early Field Trial (EFT) of
   Cisco NX-OS 7.0(3)I2(1).
+
+[git-flow]: https://github.com/petervanderdoes/gitflow-avh
+[SimpleCov]: https://github.com/colszowka/simplecov
+
+[v1.1.0]: https://github.com/cisco/cisco-network-node-utils/compare/v1.0.1...v1.1.0
```
Comment (mikewiebe) [inline CHANGELOG.md:95]: @glennmatthews This link will not work until we actually create the git tag for `v1.1.0` after collapse into master right?

Comment (glennmatthews) [inline CHANGELOG.md:95]: That's correct.

**Comment by mikewiebe** [comment]
@hunner , @jonnytpuppet , @HelenCampbell , @bmjen  Please review this release PR.  The goal is to collapse by Wed next week if possible.  Thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,13 +1,72 @@
...
+  * ntp_server (@jonnytpuppet)
+* RADIUS
+  * radius_global (@jonnytpuppet)
+  * radius_server (@jonnytpuppet)
+* SYSLOG
+  * syslog_server (@jonnytpuppet)
+  * syslog_setting (@jonnytpuppet)
+* Miscellaneous 
+  * dns_domain (@hunner)
+  * domain_name (@bmgen)
```
Comment (bmjen) [inline CHANGELOG.md]: `@bmjen` :)

**Comment by bmjen** [comment]
:+1: Overall, LGTM.

File: CHANGELOG.md (hunk 3 above)
Comment (chrisvanheuveln) [inline CHANGELOG.md]: Apologies. I blame Mike... :-)

**Pull request merged**